exhaustive.go

```diff
@@ -145,13 +152,21 @@ switch on all these enums.
 
 Skip analysis
 
-To skip checking of a specific switch statement, associate the comment shown in
-the example below with the switch statement. Note the lack of whitespace between
-the comment marker ("//") and the comment text ("exhaustive:ignore").
+In implicitly exhaustive switch mode, skip checking of a specific switch
```

Comment:
Add for clarification in parenthesis:

> (-explicit-exhaustive-switch flag off)

Something like:

> In implicitly exhaustive switch mode (-explicit-exhaustive-switch flag off), skip checking of ...

--

Here and for the next paragraph.

Reply:
I put down "(-explicit-exhaustive-switch=true)" and (-explicit-exhaustive-switch=false)". Does that suffice?